inst/WORDLIST

```diff
@@ -0,0 +1,193 @@
+AbstractCategories
```

Comment:
This appears to be copied from `crunch`. Surely most of these words don't appear here. Use `spelling::update_wordlist()` to create/update this.